website/versioned_sidebars/version-2.8-sidebars.json

```diff
@@ -1,17 +1,11 @@
 [
-  [
-    {
-      "type": "category",
-      "label": "API Client",
-      "items": [
-        "index",
-        "features",
-        "usage_concepts"
```

Comment:
why are you touching the old version snapshots? 

i would just update the current version + next, no need to deal with the older versions. 

sidenote: we should probably switch to versioning docs only for major versions as we don't really have version specific docs anyway

Reply:
ok